### PR TITLE
CE: Added deleted namespace string

### PR DESCRIPTION
**Pull request by divyaac**
### Description
When a namespace is deleted, display "deleted namespace <ns_id>" as the namespace path.
ENT pr: https://github.com/hashicorp/vault-enterprise/pull/6448

### TODO only if you're a HashiCorp employee
- [ ] **Labels:** If this PR is the CE portion of an ENT change, and that ENT change is
  getting backported to N-2, use the new style `backport/ent/x.x.x+ent` labels
  instead of the old style `backport/x.x.x` labels.
- [ ] **Labels:** If this PR is a CE only change, it can only be backported to N, so use
  the normal `backport/x.x.x` label (there should be only 1).
- [ ] **ENT Breakage:** If this PR either 1) removes a public function OR 2) changes the signature
  of a public function, even if that change is in a CE file, _double check_ that
  applying the patch for this PR to the ENT repo and running tests doesn't
  break any tests. Sometimes ENT only tests rely on public functions in CE
  files.
- [ ] **Jira:** If this change has an associated Jira, it's referenced either
  in the PR description, commit message, or branch name.
- [ ] **RFC:** If this change has an associated RFC, please link it in the description.
- [ ] **ENT PR:** If this change has an associated ENT PR, please link it in the
  description. Also, make sure the changelog is in this PR, _not_ in your ENT PR.

### CODE REVIEW

File: website/content/api-docs/system/internal-counters.mdx [hunk 1]
```diff
@@ -1105,6 +1105,10 @@ it may be up to 20 minutes delayed.
     values are `csv` and `json`. If no format is provided a default of `json`
     will be used.
 
+The response will include all child namespaces of the namespace in which the
+request was made. If some namespace has subsequently been deleted, its path will
+be listed as "deleted namespace :ID:."
```
Comment (akshya96) [inline website/content/api-docs/system/internal-counters.mdx:1110]:
```suggestion
be listed as "deleted namespace :ID:"
```

File: website/content/api-docs/system/internal-counters.mdx [hunk 2]
```diff
@@ -1058,10 +1058,10 @@ $ curl \
 ```
 
 <Note title="Important change to supported versions">
-   As of 1.16.7, 1.17.3 and later, 
-   the <a href="/vault/docs/concepts/billing-start-date">billing start date</a> automatically 
+   As of 1.16.7, 1.17.3 and later,
+   the <a href="/vault/docs/concepts/billing-start-date">billing start date</a> automatically
```
Comment (akshya96) [inline website/content/api-docs/system/internal-counters.mdx:1062]: wondering if the space that got deleted after "automatically" will delete the space in the website too. If that happens, we will have to add a space back here.

**Review by akshya96 — APPROVED** [review]
I left some comments regarding doc changes. Otherwise, it looks good to me!

File: website/content/api-docs/system/internal-counters.mdx (hunk 2 above)
Comment (divyaac) [reply website/content/api-docs/system/internal-counters.mdx:1062]: Confirmed that the space didn't get delete on the website!

**Pull request merged**